packages/react-components/src/entities/EventSidebar/details/Measurements.js

```diff
@@ -9,18 +9,23 @@ export function Measurements({ data }) {
     const [fixedColumn, setFixed] = useState(true);
 
     let hasMeasurements = false;
-    if (data.results.documents.results
-        && data.results.documents.results.length > 0
-        && data.results.documents.results[0].measurementOrFacts
-        && data.results.documents.results[0].measurementOrFacts.length > 0) {
+    if (data.documents.results
```

Comment:
Could you please use optional chaining instead. It is easier to read the intention I find. It is also what the project do in general

Reply:
Done